bundles/org.openhab.binding.luxtronikheatpump/README.md

```diff
@@ -253,7 +252,7 @@ The following channels are also writable:
 | channel  | type   | advanced | description                  |
 |----------|--------|----------|------------------------------|
 | temperatureHeatingParallelShift | Number:Temperature |   | Heating temperature (parallel shift) |
-| temperatureTargetHotWater_2 | Number:Temperature |   | Hot water temperature |
+| temperatureHotWaterTarget | Number:Temperature |   | Hot water target temperature |
```

Comment:
According to the [channel type](https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L63-L68) it is read only.

Reply:
I did some further investigation and found that the `temperatureHotWaterTarget` channel is actually defined as a writable channel [(Link)](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/enums/HeatpumpChannel.java#L1196-L1211). So the channel type seems to be incorrect.
It is interesting however that the readonly channel type of `temperatureHotWaterTarget` does not stop me from writing to it.

Tested using heatpump type `MSW 12` on openHAB `3.1.0.M4`.

Reply:
Perhaps the read only value is currently only used by the UI to disable controls for changing the value.
Do you think it is a good idea to change the channel type @sgiehl? Maybe some others also need to be fixed.

Reply:
Seems I made a mistake with those channels while renaming them in the inital PR.
The heatpump actually serves a readonly and a writable channel that both serve the same value. I thought I had removed the readonly one so only the writable is available, as having both doesn't make much sense. It's correct that the channel `temperatureTargetHotWater_2` should be removed/replaced with `temperatureHotWaterTarget`.
@martin1 could you also remove those lines: https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L63-L69
And rename the channel (id & label) here: https://github.com/openhab/openhab-addons/blob/cfea5a2185929a5a2172d7c5159232b5a2b5c5ae/bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml#L1964-L1969